osc-export/server.bndrun

```diff
@@ -85,11 +85,10 @@
 	joda-time;version='[2.9.7,2.9.8)',\
 	sdn-controller-api;version='[1.0.0,1.0.1)',\
 	security-mgr-api;version='[1.0.0,1.0.1)',\
-	org.apache.commons.collections;version='[3.2.2,3.2.3)',\
-	org.apache.commons.configuration;version='[1.10.0,1.10.1)',\
-	org.apache.commons.fileupload;version='[1.3.1,1.3.2)',\
 	org.apache.commons.codec;version='[1.6.0,1.6.1)',\
 	org.apache.commons.logging;version='[1.1.3,1.1.4)',\
 	osc-server;version='[1.0.0,1.0.1)',\
 	osc-common;version='[1.0.0,1.0.1)',\
-	log4j;version='[1.2.17,1.2.18)'
\ No newline at end of file
+	log4j;version='[1.2.17,1.2.18)',\
```

Comment:
good catch...will update it. 

The run bundles are autogenerated as part of the resolve operation so i dont think my changes effected this...probably bndrun is stale?